### DISCUSSION TITLE
tomllib module is missing

### DISCUSSION THREAD

**Discussion by ulrmehdi** [crewAIInc/crewAI#1499] [General]
Hello, I am running into a problem while setting up crewai in my conda environment in VS Code, which is running on Linux(ubuntu). My python version in this environment is 3.10, so, I should not be getting the complaints about tomllib at all, is that correct?  What are your ideas on how to fix this problem? Thanks!  Details: crewai==0.75.1
crewai-tools==0.13.2, and I see tomli=2.0.2 is a freeze, so, not sure why I need tomllib, do I?

**Comment by theCyberTech** [comment]
can you use Python 3.11?

**Reply by ulrmehdi** [reply]
Yes, it was what I did yesterday after sending my question. My initial thoughts were that I should try first a more older version of CrewAI with examples available from many places. But, I could not set up the framework properly for two agents for a couple of days. My problems were mostly related to litellm and gpt-4o model integration.  Now I moved to 3.11, and I can create a project and fill yaml files with my roles, stories and tasks. I hope I can get my framework set up promptly soon.
